CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -459,11 +459,12 @@ private static void OnAddressChanged(DependencyObject sender, DependencyProperty
 
         protected virtual void OnAddressChanged(string oldValue, string newValue)
         {
-            if (ignoreUriChange)
+            if (ignoreUriChange || newValue == null)
```

Comment:
Slight change in behavior, whilst logical, will this affect existing users? To be honest I can't remember what passing `null` does, so would have to look further.